puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemKatelloNightlyRelease.groovy

```diff
@@ -0,0 +1,36 @@
+pipeline {
+    agent { label 'admin' }
+
+    options {
+        timestamps()
+        timeout(time: 60, unit: 'MINUTES')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage ("Build Gem") {
+            steps {
+                dir('katello') {
+                    configureRVM('2.2')
```

Comment:
I generally like magic numbers to be in variables, especially if they're being reused :)

Reply:
magic number has been variablized